src/image.ts

```diff
@@ -65,18 +65,18 @@ function src(imageMappings: ImageMappings, input: string, width: number): string
         quality: defaultQuality.toString(),
         fit: 'bounds',
         'sig-ignores-params': 'true',
-        s: imageMappings[url.pathname],
+        s: sign(salt, url.pathname),
```

Comment:
So this will be pulled out from the rehydration?

Reply:
This code won't appear in the client-side bundle, it only runs on the server.